modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -143,7 +143,7 @@ pub enum SendOperationState {
     /// We are waiting for the gateway to complete the payment.
     Funded,
     /// The payment was successful.
-    Success,
+    Success { preimage: [u8; 32] },
```

Comment:
Technically this requires a DB migration, practically there's no federation with the module yet afaik or clients that support it.

Reply:
I think we had this conversation before but I forgot the outcome. These states aren't persisted in the database, they're just emitted as part of the subscription stream. They're only `Serialize/Deserialize` and not `Encodable/Decodable`.

Reply:
I was indeed confused and this is apparently the high-level state. But even the high level states are persisted to disk in case they are final ([see `caching_operation_update_stream`](https://github.com/fedimint/fedimint/blob/664955411fc1c3fd266fcaf2a9d2bd708f929736/fedimint-client/src/oplog.rs#L351)). Since these cached outcomes are serialized as JSON though we could just make the new field have a default value.

Reply:
Added default serde value